src/execution.js

```diff
@@ -86,3 +101,92 @@ export function getData(projectId, elements, executionConfiguration) {
     return d.promise();
 }
 
+const getFilterExpression = listAttributeFilter => {
+    const attributeUri = get(listAttributeFilter, 'listAttributeFilter.attribute');
+    const elements = get(listAttributeFilter, 'listAttributeFilter.default.attributeElements', []);
```

Comment:
Filter with empty elements will not be stored in MD - see https://github.com/gooddata/gdc-analytical-designer/pull/104/commits/89d82e266f18cd716296262ebfcbbf97ca3ff309